.changeset/two-badgers-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+'example-app': patch
```

Comment:
This changeset is not required since we don't publish this package!

(I hope there was a way in changeset to not show all packages when `yarn changeset` is run. They do have an `ignored` config, but it doesn't really work for the prompt.

Reply:
@OrkoHunter I will delete the file then before merging the PR :)